src/components/modules/banners/contributions/ContributionsBannerWrapper.tsx

```diff
@@ -99,28 +79,60 @@ const withBannerData = (
             numArticles,
             'banner',
         );
-        const mobileMessageTextWithArticleCount = !!cleanMobileMessageText
-            ? replaceArticleCount(cleanMobileMessageText, numArticles, 'banner')
-            : null;
         const highlightedTextWithArticleCount = !!cleanHighlightedText
             ? replaceArticleCount(cleanHighlightedText, numArticles, 'banner')
             : null;
 
-        if (!copyHasPlaceholder) {
+        if (copyHasPlaceholder) {
+            throw Error('Banner copy contains placeholders, abandoning.');
+        }
+
+        return {
+            cleanHighlightedText: highlightedTextWithArticleCount,
+            cleanMessageText: messageTextWithArticleCount,
+            cleanHeading: headingWithArticleCount,
+            ctaUrl,
+            ctaText: cta.text,
```

Comment:
Not a big one - naming is maybe a tad unclear here. We have a `cleanHighlightedText` above where clean = no non article count placeholders, but clean here means article counts have been subbed out too.

Reply:
True. Perhaps in this model it should just be `heading`, `messageText` etc, since this is the `ContributionsBannerRenderedContent`

Reply:
Yeah that would make sense. If that's too much effort though, I think it's already okay